locale/en.yml

```diff
@@ -4,6 +4,13 @@ flarum-mentions:
   # UNIQUE KEYS - The following keys are used in only one location each.
   ##
 
+  # Translations in this namespace are used by the admin interface.
+  admin:
+
+    # These translations are used in the mentions Settings page.
+    settings:
+      allow_username_format: Allow username mention format
```

Comment:
Can we add a "help" item in addition to the label with an example of the difference between the formats?